Car.cpp

```diff
@@ -0,0 +1,35 @@
+#include "Car.hpp"
+
+#include <iostream>
+
+#include "InvalidSpeed.hpp"
+
+Car::Car() {
+    std::cout << __FUNCTION__ << std::endl;
+}
+
+Car::~Car() {
+    std::cout << __FUNCTION__ << std::endl;
+}
+
+void Car::turnLeft() {
+    std::cout << __FUNCTION__ << std::endl;
+}
+
+void Car::turnRight() {
+    std::cout << __FUNCTION__ << std::endl;
+}
+
+void Car::brake() {
+    std::cout << __FUNCTION__ << std::endl;
+    setSpeed(MINIMUM_VELOCITY);
+}
+
+void Car::accelerate(int speed) {
+    if (speed < MINIMUM_VELOCITY) {
```

Comment:
What flags are you using during compiling? You are comparing here int with bool. I would be suprised if compilator with all flags would not generate warning here.

Reply:
Honestly, this variable was supposed to be uint8_t xd. Im using g++ 9.3.0

Reply:
I was asking about flags(like -wall -pedantic). It should warn you that you are comparing different types.